src/System.Security.Cryptography.Encryption/src/System/Security/Cryptography/CryptoStream.cs

```diff
@@ -1,31 +1,29 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using System;
+using System.Diagnostics.Contracts;
 using System.IO;
 using System.Runtime.InteropServices;
-using System.Diagnostics.Contracts;
-using System.Threading;
-using System.Threading.Tasks;
+
 
 namespace System.Security.Cryptography
 {
-    [System.Runtime.InteropServices.ComVisible(true)]
+    [ComVisible(true)]
```

Comment:
Nope.  I previously scrubbed out a bunch of these.  I'll make another pass to do so again, as some more have snuck in (like this one).